scripts/write_srl_predictions_to_conll_format.py

```diff
@@ -19,7 +19,7 @@ def main(serialization_directory, device):
         The device to run the evaluation on.
     """
 
-    config = Params.from_file(os.path.join(serialization_directory, "model_params.json"))
```

Comment:
If you could make this backward compatible that would be great, maybe just with a `try except` or something.